.devcontainer/post-create.sh

```diff
@@ -22,8 +22,6 @@ psql -U admin -d aoai-proxy -h localhost -w -f ./database/aoai-proxy.sql
 
 echo Setting up Python environment...
 
-python3 -m pip install -r requirements-dev.txt
-
```

Comment:
Won't we need that so that `pre-commit` is installed?

Reply:
No, I'm confident they were package requirements for Python Proxy only. The Python runtime still being install... Besides, the requirements-dev.txt was in the proxy folder which is now removed...